testing/xml_testing/sort.go

```diff
@@ -0,0 +1,69 @@
+package xml_testing
```

Comment:
Suggest naming this `xmltesting` without the `_`

Reply:
Probably also want to move the `smithy-go/testing#AssertXML` and `XMlEqual` functions to this package as well.